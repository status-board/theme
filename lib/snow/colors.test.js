"use strict";

var _colors = _interopRequireDefault(require("./colors"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

test('Colors should match snapshot', function () {
  expect(_colors["default"]).toMatchSnapshot();
});
//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9zbm93L2NvbG9ycy50ZXN0LnRzIl0sIm5hbWVzIjpbInRlc3QiLCJleHBlY3QiLCJjb2xvcnMiLCJ0b01hdGNoU25hcHNob3QiXSwibWFwcGluZ3MiOiI7O0FBQUE7Ozs7QUFFQUEsSUFBSSxDQUFDLDhCQUFELEVBQWlDLFlBQVk7QUFDN0NDLEVBQUFBLE1BQU0sQ0FBQ0Msa0JBQUQsQ0FBTixDQUFlQyxlQUFmO0FBQ0gsQ0FGRyxDQUFKIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IGNvbG9ycyBmcm9tICcuL2NvbG9ycyc7XG5cbnRlc3QoJ0NvbG9ycyBzaG91bGQgbWF0Y2ggc25hcHNob3QnLCAoKTogdm9pZCA9PiB7XG4gICAgZXhwZWN0KGNvbG9ycykudG9NYXRjaFNuYXBzaG90KCk7XG59KTtcbiJdfQ==