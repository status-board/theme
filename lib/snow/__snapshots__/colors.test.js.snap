// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors should match snapshot 1`] = `
Object {
  "background": "#ebf0f1",
  "failure": "#e37856",
  "icon": "#0badc2",
  "success": "#8ddb8d",
  "text": "#333",
  "unknown": "#d3dfe8",
  "warning": "#d1be65",
}
`;
